src/odemis/gui/cont/acquisition.py

```diff
@@ -1669,6 +1669,7 @@ def __init__(self, tab_data, tab_panel):
         self._tab_data_model = tab_data
         self._main_data_model = tab_data.main
         self._tab_panel = tab_panel
+        self._scanner = tab_data.main.ebeam
```

Comment:
is this really necessary? Why not change line 1706 to:
`self._tab_data_model.main.ebeam.dwellTime.subscribe(self.update_acquisition_time)`

Reply:
Not really necessary. Thought it looks cleaner. I changed it.